src/corppa/poetry_detection/annotation/process_adjudication_data.py

```diff
@@ -147,24 +120,29 @@ def process_adjudication_data(
     )
     csv_fieldnames = [
         "page_id",
-        "work_id",
-        "work_title",
-        "work_author",
-        "work_year",
-        "start",
-        "end",
-        "text",
+        "excerpt_id",
+        "ppa_span_start",
+        "ppa_span_end",
+        "ppa_span_text",
+        "detection_methods",
+        "notes",
     ]
     with open(output_excerpts, mode="w", newline="") as csvfile:
         csv_writer = csv.DictWriter(csvfile, fieldnames=csv_fieldnames)
         csv_writer.writeheader()
         for page_anno in progress_annos:
-            # Get & save page data
             page_data = process_page_annotation(page_anno)
-            orjsonl.append(output_pages, page_data)
 
-            for row in get_excerpt_entries(page_data):
-                csv_writer.writerow(row)
+            # Write excerpt-level data
+            print(page_data)
```

Comment:
In case you forgot, you left a print statement in (noticed when I ran the script just now).